docs/customizing_deis/choosing-a-scheduler.rst

```diff
@@ -0,0 +1,143 @@
+:title: Choosing a Scheduler
+:description: How to choose a scheduler backend for Deis.
+
+
+.. _choosing_a_scheduler:
+
+Choosing a Scheduler
+====================
+
+The scheduler creates, starts, stops, and destroys your app containers. For example,
+a command such as ``deis scale web=3`` tells the scheduler to run three containers
```

Comment:
we should reference [Container](http://docs.deis.io/en/latest/reference/terms/container/) here as a defined term. We should probably do the same here for a scheduler.